docs/getting_started.md

```diff
@@ -1,2 +1,32 @@
 # Getting Started
```

Comment:
I wonder if changing the language here to "How to create a new app" would be better here? I could see us also wanting a "getting started developing" section for new devs coming on to an existing project covering some of the stuff we learned from Apple yesterday.

Reply:
I'm not sure about "How to create a new app" as I think that implies something more technical? Like creating a new app in Xcode or Android Studio. I dunno though.

I envisioned this page to discuss what is needed at the onboarding meeting. That includes the info needed to get setup on the Apple and Google stores. Plus finding out what the team is building and if they have the resources in place.

Maybe something like:
* Onboard Meeting
* Project Initiation
* Kickoff meeting
* Something else

Thoughts?

Reply:
renamed to "Project initiation"